global/data/example-data.json

```diff
@@ -1,5 +1,5 @@
 {
-    "_migration_index": 21,
+    "_migration_index": 26,
```

Comment:
@r-peschke it seems like the migration index of the example data is checked nowhere